README.md

```diff
@@ -73,7 +73,7 @@ Here a configuration example:
 
 ```yaml
 m6_web_amqp:
-    sandbox: false
+    sandbox: false #optional - default false
```

Comment:
There was no notice of static configuration. With such request feature explodes in code size. 
If you have certain vision about this in memory queue - feel free to add it